apps/web/src/app/(pages)/(dashboard)/roles/page.tsx

```diff
@@ -33,10 +53,57 @@ export default function Roles({
     reviews.data ? reviews.data[0] : undefined,
   );
 
+  const [showAlert, setShowAlert] = useState(true);
+
   return (
     <>
       <SearchFilter />
       {/* TODO: Loading animations */}
+      {error && showAlert && (
+        <div
```

Comment:
Totally optional but I'd recommend taking a look at [shadcn/ui](https://ui.shadcn.com/). Almost all of our reusable components are built on top of shadcn. They do have a [toast](https://ui.shadcn.com/docs/components/toast) that you can use! 

If you're interested, check out [this](https://cooper-docs.vercel.app/docs/web/ui) docs page. You can add the component from the root folder using `pnpm ui-add toast` and use that component here.